pyproject.toml

```diff
@@ -110,7 +110,13 @@ YouTube = "https://www.youtube.com/qctrl"
 GitHub = "https://github.com/qctrl"
 
 # Uncomment the block below if the package requires access to the Internal PyPI.
```

Comment:
```suggestion
# Uncomment the following 2 blocks if the package requires access to the Internal PyPI.
```
Just to make it very clear that you also need to uncomment the PyPI as 'supplemental' configuration block.